src/adapters/polkadot.ts

```diff
@@ -153,23 +153,13 @@ class BasePolkadotAdapter extends BaseCrossChainAdapter {
     }
 
     return combineLatest({
-      txFee: this.estimateTxFee({
-        amount: FN.ZERO,
-        to,
-        token,
-        address,
-        signer: address,
-      }),
       balance: this.balanceAdapter
         .subscribeBalance(token, address)
         .pipe(map((i) => i.available)),
     }).pipe(
-      map(({ balance, txFee }) => {
-        const tokenMeta = this.balanceAdapter?.getToken(token);
-        const feeFactor = 1.2;
-        const fee = FN.fromInner(txFee, tokenMeta?.decimals).mul(
-          new FN(feeFactor)
-        );
+      map(({ balance }) => {
+        // fixed fee of 1 ksm or DOT until we get paymentinfo to work again
+        const fee = FN.fromInner(Math.pow(10, tokenMeta?.decimals), tokenMeta?.decimals);
```

Comment:
```suggestion
        // fixed fee of 0.05 ksm or DOT until we get paymentinfo to work again
        const fee = FN.fromInner(
          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
          0.05 * Math.pow(10, tokenMeta!.decimals),
          tokenMeta?.decimals
        );
```

Reply:
Also need to rename the (currently unused) `to` parameter: `_to` . 
to get rid of linter error.